src/python/library/tritonclient/grpc/__init__.py

```diff
@@ -216,8 +219,9 @@ def __init__(self,
             ('grpc.http2.max_pings_without_data',
                 keepalive_options.http2_max_pings_without_data),
         ]
-
-        if ssl:
+        if creds:
+            grpc.secure_channel(url, creds, options=channel_opt)
```

Comment:
Did you validate your changes?
Shouldn't this be:

```
self._channel = grpc.secure_channel(url, creds, options=channel_opt)
```

Reply:
Sorry that for the hasty pr, I've just fixed the issues and tested the changes